lib/connection.ts

```diff
@@ -29,6 +30,15 @@ export interface SenderOptionsWithSession extends SenderOptions {
   session?: Session;
 }
 
+/**
+ * Describes the options that can be provided while creating an asynchronous AMQP sender.
+ * One can also provide a session if it was already created.
+ * @interface AsynchronousSenderOptionsWithSession
+ */
+export interface AsynchronousSenderOptionsWithSession extends SenderOptions {
```

Comment:
Shouldnt this be extending `AsynchronousSenderOptions`?

Reply:
Oh yes! Will change this.